__tests__/app.test.js

```diff
@@ -100,6 +100,49 @@ describe('GET:', () => {
 					expect(body.msg).toEqual('Not found');
 				});
 			})
+			test('status:200 should sort by given category defaulting to descending order', () => { 
+				return request(app)
+				.get('/api/articles?topic=mitch&sort_by=title')
+				.expect(200)
+					.then(res => {
+						expect(res.body.length).toBe(4)
+						expect(res.body).toBeSortedBy('title', {descending: true});
+				});
+			})
+			test('status:200 should sort by given category in descending order when specified', () => {
+				return request(app)
+				.get('/api/articles?topic=mitch&sort_by=title&order=des')
+				.expect(200)
+					.then(res => {
+						expect(res.body.length).toBe(4)
+						expect(res.body).toBeSortedBy('title', {descending: true});
+				});
+			})
+			test('status:200 should sort by given category in ascending order when specified', () => { 
+				return request(app)
+				.get('/api/articles?topic=mitch&sort_by=title&order=asc')
+				.expect(200)
+					.then(res => {
+						expect(res.body.length).toBe(4)
+						expect(res.body).toBeSortedBy('title', {descending: false});
+				});
+			})
+			test('status:200 should return sort order as descending if order value is invalid', () => { 
+				return request(app)
+				.get('/api/articles?sort_by=title&order=apple')
+				.expect(200)
+				.then(res => {
+					expect(res.body).toBeSortedBy('title', {descending: true});
+			});
```

Comment:
I like this approach, my notes suggest to return an error here as the order is invalid. However I like the idea of just defaulting and ignoring the invalid order. Up to you how you'd like to handle it!